goss/goss-mysql.yaml

```diff
@@ -0,0 +1,20 @@
+service:
+  mysql:
+    enabled: true
+    running: true
```

Comment:
not sure if the `service` field has been auto-generated but mysql runs in the docker container as pid 1 and doesn't use a service manager.

Reply:
Yup, I generated this with the Goss autoadd, I'll find an alternative